cpp/src/IceDiscovery/Makefile.mk

```diff
@@ -5,5 +5,6 @@ $(project)_libraries := IceDiscovery
 IceDiscovery_targetdir                  := $(libdir)
 IceDiscovery_dependencies               := Ice
 IceDiscovery_cppflags                   := -DICE_DISCOVERY_API_EXPORTS $(api_exports_cppflags)
+IceDiscovery_sliceflags                 := --include-dir IceDiscovery
```

Comment:
I could not figure out how to generate the header next to the .cpp file like somehow we do in IceGrid.